min_version_test/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: '2.17.0'
+  sdk: '>=2.17.1 <4.0.0'
```

Comment:
The `min_version_test` project is specifically created to test against the lowest supported SDK version (2.17.0), so this change doesn't seem right. Why is it necessary?

Reply:
Fixed